src/main/resources/application.conf

```diff
@@ -108,18 +108,18 @@ database {
 
   main {
     hsqldb {
-      url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
-      driver = "org.hsqldb.jdbcDriver"
-      slick.driver = "slick.driver.HsqldbDriver"
+      db.url = "jdbc:hsqldb:mem:${slick.uniqueSchema};shutdown=false;hsqldb.tx=mvcc"
+      db.driver = "org.hsqldb.jdbcDriver"
+      driver = "slick.driver.HsqldbDriver$"
```

Comment:
The built in slick drivers are [all][1] instances of scala's singleton `object`. I was using [scala][2] [reflection][3] API for retrieving the singleton. Slick's built in [`DatabaseConfig`][4] chose to use a [short][5] [cut][6](imho), but it use the Java reflection API, where the companion object has a class name that ends with a `$`.

[1]: https://github.com/slick/slick/blob/3.1.0/slick/src/main/scala/slick/driver/MySQLDriver.scala#L280
[2]: http://stackoverflow.com/questions/23466782/scala-object-get-reference-from-string-in-scala-2-10
[3]: https://github.com/anvie/slick-test/blob/045f4db610d3b91bf928a53f2bc7b6ae17c35985/slick-util/src/main/scala/scala/slick/codegen/ModelGenerator.scala
[4]: https://github.com/slick/slick/blob/3.1.0/slick/src/main/scala/slick/backend/DatabaseConfig.scala#L41-L43
[5]: http://www.scala-lang.org/old/node/7065
[6]: https://github.com/slick/slick/blob/3.1.0/slick/src/main/scala/slick/backend/DatabaseConfig.scala#L62-L63